.gitlab-ci.yml

```diff
@@ -215,6 +215,20 @@ run-tests:
     - job: image-build-tests-runner
     - job: image-build-runner
 
+boardfarm-tests:
+  stage: test
+  image: $CI_REGISTRY_IMAGE/prplmesh-boardfarm-ci:$CI_PIPELINE_ID
+  script:
+    - export USER=gitlab-runner BFT_DEBUG=y
```

Comment:
These exports should be explained in a comment or in the commit message.

Reply:
Done.